_docs/technical/2017-01-04-updater.md

```diff
@@ -111,23 +120,27 @@ Please refer to the [core types
 module](https://github.com/input-output-hk/cardano-sl/blob/22360aa45e5dd82d0c87872d8530217fc3d08f4a/src/Pos/Update/Core/Types.hs)
 for more information.
 
-## Update Proposal Adoption
+## Update Proposal Approval
 
 A very important part of implementation of the update mechanism is
 the part that works with genesis blocks for epochs and applies updates.
 This logic resides
 [in this well-documented function](https://github.com/input-output-hk/cardano-sl/blob/22360aa45e5dd82d0c87872d8530217fc3d08f4a/src/Pos/Update/Poll/Logic/Softfork.hs#L67).
 We explain terminology related to this process below.
 
-### `softforkResolutionThreshold` Predicate
+### Threshold
+
+`softforkResolutionThreshold` is a threshold (referred to as «threshold»
+in the code) which, for an update proposal with version `v`, says ???
+
+[//]: TODO: IDK how to link threshold & predicate & block version here
```

Comment:
@ChShersh It might simply be better not to include sections for which we still don't know what to write. Either that, or instead of `???`, it is written "Pending" or something, at least until we figure it out. I suggest we don't wait more than a sprint (and that's stretching it - ideally, it wouldn't be more than a day or so) before filling it in, or removing it completely. @gromakovsky can you lend a hand here as well?